src/toast/utils.py

```diff
@@ -36,6 +36,56 @@
 )
 
 
+def set_numba_openmp():
+    have_numba_omp = False
```

Comment:
what about doing `threading = "default"`,

then try omp and and set `threading="omp"` if it is available, in the expect block try with tbb.

finally `config.THREADING_LAYER = layer`

Reply:
I was just copying from the numba unit tests, but sure, I can do a nested try/except if you want.